OmiseSwift/APIClient.swift

```diff
@@ -36,12 +36,22 @@ public class APIClient: NSObject {
     func preferredKeyForServerEndpoint(_ endpoint: ServerEndpoint) -> String {
         switch endpoint {
         case .api:
-            return config.secretKey
+            return config.secretKey + ":X"
         case .vault:
             return config.publicKey
         }
     }
     
+    func preferredEncodedKeyForServerEndpoint(_ endpoint: ServerEndpoint) throws -> String {
+        let data = preferredKeyForServerEndpoint(endpoint).data(using: .utf8, allowLossyConversion: false)
```

Comment:
I think should append `:X` here and make it works the same w public/secret key na.